packages/wdio-protocols/protocols/saucelabs.json

```diff
@@ -199,5 +209,25 @@
         "description": "Returns true if any metrics have a regression."
       }
     }
+  },
+  "/session/:sessionId/sauce/ondemand/performance/scroll": {
+    "POST": {
+      "command": "jankinessCheck",
+      "description": "Perform a scroll test that evaluates the jankiness of the application.",
+      "ref": "https://wiki.saucelabs.com/display/DOCS/Custom+Sauce+Labs+WebDriver+Extensions+for+Network+and+Log+Commands",
```

Comment:
Can't see nothing here about scroll performance. Is it going to be added soon?

Reply:
yes